doc/techref/text_formatting.md

```diff
@@ -0,0 +1,21 @@
+# GMT Text Formatting
+
+The table below shows how text add to a plot as well as labels of colorbars and
+Cartesian axes can be formatted. Changing the font as well as its color and size
+in this way allows to change those only for specific words of a longer text. The
+supported fonts are listed at [](/techref/fonts.md). For Greek letters, is there
+any official list for the assignment ??? .
```

Comment:
> For Greek letters, is there any official list for the assignment ??? .

What do you mean here? `@~` is equivalent to `@%12%`

Reply:
Partly I find it not intuitive which letter refers to a desired Greek letter, e.g., `@~a@~` gives $\alpha$, but `@~j@~` gives $\varphi$. Thus, I made an overview for myself (https://github.com/yvonnefroehlich/gmt-pygmt-plotting/blob/main/000_general_stuff/05_greek_letters).
While working on this PR, I was wondering if there is any "trick" or overview in the GMT docs to figure this out. But so far I could not find something. But considering the support of encodings and non-ASCII characters (https://www.pygmt.org/dev/techref/encodings.html), this may not be so important.

Reply:
> if there is any "trick" or overview in the GMT docs to figure this out.

I think no. Usually, users should use something like `@~\141@~` to get $\alpha$.


> But considering the support of encodings and non-ASCII characters (https://www.pygmt.org/dev/techref/encodings.html), this is maybe not so important.

I agree.